pep/app/constants/dimensions.ts

```diff
@@ -1,6 +1,6 @@
 export const NAV_BAR_HEIGHT = 58;
 export const FOOT_BAR_HEIGHT = 40;
-export const SIDEBAR_WIDTH = 300;
+export const SIDEBAR_WIDTH = 315;
```

Comment:
pretty sure this needs to stay at 300px as the default size, at least for mobile, so that the sidebar toggle/nub is still visible on puny 320px wide phones, otherwise it'll be pushed off screen and they won't be able to close the sidebar. 

![Screen Shot 2021-04-30 at 10 08 39 AM](https://user-images.githubusercontent.com/611458/116707077-47d3fe80-a99c-11eb-9351-ea0e338d13f0.png)